Packages/com.unity.inputsystem/InputSystem/InputSystem.cs

```diff
@@ -2409,9 +2409,9 @@ private static void PerformDefaultPluginInitialization()
             Switch.SwitchSupport.Initialize();
             #endif
 
-            #if UNITY_EDITOR || UNITY_STANDALONE || UNITY_ANDROID || UNITY_IOS || UNITY_WSA
+            //#if UNITY_EDITOR || UNITY_STANDALONE || UNITY_ANDROID || UNITY_IOS || UNITY_WSA || UNITY_LUMIN
```

Comment:
Why is this commented out?

Reply:
Reverted it back, was just some WIP changes